app/Jobs/FetchAd.php

```diff
@@ -37,28 +45,27 @@ public function __construct(string $sourceName)
      * @param AdItemInserter $adItemInserter
      * @param AdPicTransferrer $adPicTransferrer
      */
-    public function handle(AdFetcher $adFetcher , AdItemInserter $adItemInserter , AdPicTransferrer $adPicTransferrer)
+    public function handle(AdFetcher $adFetcher, AdItemInserter $adItemInserter, AdPicTransferrer $adPicTransferrer)
     {
         $this->adFetcher = $adFetcher;
-        $this->adItemInserter=$adItemInserter;
-        $this->adPicTransferrer=$adPicTransferrer;
+        $this->adItemInserter = $adItemInserter;
+        $this->adPicTransferrer = $adPicTransferrer;
 
```

Comment:
set adFetcher, adItemInserter, adPicTransferrer in counstructor via Ioc Container(like handle method)

Reply:
done